snippets/react-snippets.code-snippets

```diff
@@ -12,6 +12,16 @@
         ],
         "description": "Adds aria attributes for a dialog. A dialog is a window separate from the rest of the webpage like a modal."
     },
+    "SearchAria":{
+        "prefix": [
+            "SearchAria",
+        ],
+        "body": [
+            "role='search'",
+            "role='searchbox'",
```

Comment:
I am sorry! I meant as a separate Snippet

SearchAria is perfect but also "SearchboxAria"

for SearchBox i would consider also looking at ComboxboxAria -> combox is a an input as well so they might have similar aria attributes